src/web/routes/teams/types.ts

```diff
@@ -5,50 +5,53 @@ import type {
 	FigmaTeamSummary,
 } from '../../../domain/entities';
 
-export type ConfigureFigmaTeamRequestBody = {
+export type ConnectFigmaTeamRequestBody = {
```

Comment:
**suggestion:** Currently, the name of request/response types do not match the names of the endpoints.

I am wondering, maybe, the idea to use `PUT /teams/:teamId/connect` and `PUT /teams/:teamId/disconnect` as endpoints was not too bad. What do you think, @jv3nk ?

Reply:
Good point, I'll think about it overnight and come back to this in the morning. I'm not 100% sure about using `PUT`, since we'd be creating a new webhook subscription for each call, so I'm not sure if I'd consider it to be fully idempotent.